Doc/library/shutil.rst

```diff
@@ -177,7 +177,7 @@ Directory and files operations
 .. function:: copy2(src, dst, *, follow_symlinks=True)
 
    Identical to :func:`~shutil.copy` except that :func:`copy2`
-   also attempts to preserve all file metadata.
+   also attempts to preserve more file metadata.
```

Comment:
I suggest to remove "more".

Reply:
Problem is that shutil.copy() still copies the mode, so it is not true that the difference is that "copy2  attempts to preserve file metadata", as copy also preserves metadata.
Anyway, I removed the 'more' word as you suggested.